run/era5_topaz4_maker.py

```diff
@@ -170,44 +171,38 @@ def topaz4_source_file_name(unix_time, path):
     
     era5_out_file = f"{filepfx}ERA5_{args.start}_{args.stop}.nc"
     print(f"Writing ERA5 data to {era5_out_file}")
-    era_root = netCDF4.Dataset(era5_out_file, "w", format="NETCDF4")
-    structgrp = era_root.createGroup("structure")
-    structgrp.type = target_structure
+    era_ncFile = netCDF4.Dataset(era5_out_file, "w", format="NETCDF4")
```

Comment:
```suggestion
    era5_ncFile = netCDF4.Dataset(era5_out_file, "w", format="NETCDF4")
```

Right? See also similar comment below.

Reply:
Thanks! I see you already fixed this.